lib/core-generators/new/templates/views/pages/legal/privacy.ejs

```diff
@@ -38,17 +38,36 @@
+    <p>If you are in the European Union, you have rights under the General Data Protection Regulation in relation to your personal data:</p>
+    <ul>
+      <li><strong>The right to access</strong> - You have the right to request NEW_APP_COMPANY_NAME for copies of your personal data. We may charge you a small fee for this service.</li>
+      <li><strong>The right to rectification</strong> - You have the right to request that NEW_APP_COMPANY_NAME correct any information you believe is inaccurate. You also have the right to request NEW_APP_COMPANY_NAME to complete the information you believe is incomplete.</li>
+      <li><strong>The right to erasure</strong> - You have the right to request that NEW_APP_COMPANY_NAME erase your personal data, under certain conditions.</li>
+      <li><strong>The right to restrict processing</strong> - You have the right to request that NEW_APP_COMPANY_NAME restrict the processing of your personal data, under certain conditions.</li>
+      <li><strong>The right to object to processing</strong> -  You have the right to object to NEW_APP_COMPANY_NAME’s processing of your personal data, under certain conditions.</li>
+      <li><strong>The right to data portability</strong> - You have the right to request that NEW_APP_COMPANY_NAME transfer the data that we have collected to another organization, or directly to you, under certain conditions.</li>
+    </ul>
+    <p>If you make a request, we have one month to respond to you. If you would like to exercise any of these rights, please <a href="/contact">contact us</a>.</p>
+    <p>Should you wish to report a complaint or if you feel that NEW_APP_COMPANY_NAME has not addressed your concern in a satisfactory manner, you have the right to lodge a complaint with your local data protection authority.</p>
 
     <h3>Electronic mailing lists and how to unsubscribe</h3>
     <p>We may use your contact information periodically in order to send you email or other communications regarding updates to the Service and to alert you or remind you of happenings related to the Service. The frequency of these messages will vary depending upon various factors.</p>
     <p>If you no longer wish to receive e-mails from us and want to be removed from our electronic mailing list, please <a href="/contact">contact us</a> and include “Email Unsubscribe” in the body of your message.  If you choose to unsubscribe from our mailing lists, we will hold your contact information on file marked so that we do not inadvertently contact you if your details are subsequently provided to us by a third party.</p>
 
+    <h3>Data protection officer</h3>
+    <p>To communicate with our Data Protection Officer, please email <%= sails.config.custom.internalEmailAddress %>.</p>
```

Comment:
Would you also update the boilerplate comments in generated `config/custom.js` and `config/env/production.js` files to clarify that, if you're using the default privacy policy, then `sails.config.custom.internalEmailAddress` will be used as the contact email of your "data protection officer" for the purpose of compliance with regulations such as GDPR?